src/libraries/System.Text.Json/tests/Serialization/CyclicTests.cs

```diff
@@ -21,35 +21,43 @@ public static void WriteCyclicFailDefault()
         }
 
         [Theory]
-        [InlineData(1)]
-        [InlineData(2)]
-        [InlineData(10)]
-        [InlineData(70)]
-        public static void WriteCyclicFail(int depth)
+        [InlineData(1, 2)]
+        [InlineData(2, 3)]
+        [InlineData(10, 11)]
+        [InlineData(70, 71)]
+        [InlineData(10, 0)]
```

Comment:
nit: Add a comment here that 0 (or default) max depth is treated as 64. Something like this (feel free to reword):
```suggestion
        [InlineData(10, 0)] // 0 (or default) max depth is treated as 64.
```

Reply:
fixed in 70835c4